microprofile/WFLY-11529_subsystem_metrics.asciidoc

```diff
@@ -0,0 +1,145 @@
+= [WFLY-11529] Expose WildFly Metrics
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          observability,microprofile,metrics,openshift
+
+== Overview
+
+The goal of this feature is to improve observability of WildFly by exposing its own metrics to monitoring tools such as https://prometheus.io[Prometheus]
+and the https://docs.openshift.com/container-platform/3.11/architecture/infrastructure_components/web_console.html[OpenShift Console].
+
+This feature exposes the WildFly metrics (from the subsystem and deployment management models) to the `/metrics` HTTP endpoint in the Prometheus format.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11529[WFLY-11529]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1169[EAP7-1169]
```

Comment:
change this line to
* https://issues.jboss.org/browse/EAP7-1169[EAP7-1169] - EAP7-1169 Expose EAP subsystem Metrics

to be in sync with the line 26